src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
-import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
```

Comment:
same here with name of type- choose more descriptive 😉

Reply:
done!